File: jhipster/angular-microservices/scheduler/src/main/java/com/stpl/jobscheduler/domain/CronEntity.java
package com.stpl.jobscheduler.domain;

public class CronEntity {
    private String cronType;
    private DailyFre dailyFrq;
    private WeeklyFre weeklyFrq;
    private MonthlyFre monthlyFrq;
    private CustomFre customFrq;

    /**
     * @return the dailyFrq
     */
    public DailyFre getDailyFrq() {
        return dailyFrq;
    }

    /**
     * @return the customFrq
     */
    public CustomFre getCustomFrq() {
        return customFrq;
    }

    /**
     * @param customFrq the customFrq to set
     */
    public void setCustomFrq(CustomFre customFrq) {
        this.customFrq = customFrq;
    }

    /**
     * @return the monthlyFrq
     */
    public MonthlyFre getMonthlyFrq() {
        return monthlyFrq;
    }

    /**
     * @param monthlyFrq the monthlyFrq to set
     */
    public void setMonthlyFrq(MonthlyFre monthlyFrq) {
        this.monthlyFrq = monthlyFrq;
    }

    /**
     * @return the cronType
     */
    public String getCronType() {
        return cronType;
    }

    /**
     * @param cronType the cronType to set
     */
    public void setCronType(String cronType) {
        this.cronType = cronType;
    }

    /**
     * @param dailyFrq the dailyFrq to set
     */
    public void setDailyFrq(DailyFre dailyFrq) {
        this.dailyFrq = dailyFrq;
    }

    /**
     * @return the weeklyFrq
     */
    public WeeklyFre getWeeklyFrq() {
        return weeklyFrq;
    }

    /**
     * @param weeklyFrq the weeklyFrq to set
     */
    public void setWeeklyFrq(WeeklyFre weeklyFrq) {
        this.weeklyFrq = weeklyFrq;
    }
}

File: jhipster/angular-microservices/scheduler/src/main/webapp/app/model/CronEntity.ts
export class CronEntity {
    cronType: String;
    dailyFrq: DailyFre;
    weeklyFrq: WeeklyFre;
    monthlyFrq: MonthlyFre;
    customFrq: CustomFre;
}

export class DailyFre{
    hour: number;
    min: number;
    sec: number;
}

export class WeeklyFre {
    weekDay: String;
    hour: number;
    min: number;
    sec: number;
}

export class MonthlyFre {
    date: number;
    hour: number;
    min: number;
    sec: number;
}

export class CustomFre {
    customDate: Date;
    hour: number;
    min: number;
    sec: number;
}